src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/SentenceValidationService.kt

```diff
@@ -98,10 +98,10 @@ class SentenceValidationService(
 
   private fun validateDuration(sentencesAndOffence: SentenceAndOffence): List<ValidationMessage> {
     val sentenceCalculationType = SentenceCalculationType.from(sentencesAndOffence.sentenceCalculationType)
-    return if (sentenceCalculationType.sentenceClazz == StandardDeterminateSentence::class.java ||
-      sentenceCalculationType.sentenceClazz == AFineSentence::class.java ||
-      sentenceCalculationType.sentenceClazz == DetentionAndTrainingOrderSentence::class.java ||
-      sentenceCalculationType.sentenceClazz == BotusSentence::class.java
+    return if (sentenceCalculationType.sentenceType.sentenceClazz == StandardDeterminateSentence::class.java ||
```

Comment:
This should probably be refactored into something that describes which types are singleTermAble?

Reply:
I've done this refactoring, but might be worth taking a look at ...